### PR TITLE
Re: #1254 - Track links to non-webpage URLs in MOI Splash Page

### PR DESCRIPTION
**Pull request by mmmavis**
@adamlofting, could you please review this PR for me? Is delaying for 1 sec enough?

### CODE REVIEW

File: public/js/pages/localweb.js [hunk 1]
```diff
@@ -20,9 +20,35 @@ requirejs.config({
...
   "use strict";
   var $window = $(window);
+  var TIME_DELAYED_REDIRECT = 1000; // in milliseconds
+  // analytics
+  var delayRedirect = function(url) {
+    setTimeout(function() {
+      window.location.href = url;
+    }, TIME_DELAYED_REDIRECT);
+  };
+  $("#header-partner-with-us-link").click(function() {
```
Comment (mmmavis) [inline public/js/pages/localweb.js]: this element now links to a same page anchor, not sure if we want to record this?

File: public/js/pages/localweb.js [hunk 2]
```diff
@@ -20,9 +20,35 @@ requirejs.config({
...
+  });
+  $("#moi-video-play-btn").click(function() {
+    analytics.event("Play Video", {label: "MOI Partner Opportunity"});
+  });
+  $(".report-links").click(function(event) {
+    event.preventDefault();
+    analytics.event("Download Report", {label: $(this).data("report-name")});
+    delayRedirect($(this).attr("href"));
+  });
+  $("#mobile-webmaker-partners-link").click(function() {
```
Comment (mmmavis) [inline public/js/pages/localweb.js:44]: This wasn't mentioned in this [ticket](https://github.com/mozilla/webmaker.org/issues/1254) but I guess that's cuz copy and specs got changed again after that ticket was created :)

File: public/js/pages/localweb.js [hunk 3]
```diff
@@ -20,9 +20,35 @@ requirejs.config({
   }
 });
 
-require(["jquery", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, languages, selectize ) {
+require(["jquery", "analytics", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, analytics, languages, selectize ) {
   "use strict";
   var $window = $(window);
+  var TIME_DELAYED_REDIRECT = 1000; // in milliseconds
```
Comment (mmmavis) [inline public/js/pages/localweb.js]: is 1 sec enough?

Comment (adamlofting) [inline public/js/pages/localweb.js]: we can make this 500ms so it's a bit more responsive

File: public/js/pages/localweb.js (hunk 1 above)
Comment (adamlofting) [inline public/js/pages/localweb.js]: No, let's remove this need if we're just linking to the anchor.

File: public/js/pages/localweb.js (hunk 2 above)
Comment (adamlofting) [inline public/js/pages/localweb.js:44]: I can't see a `#mobile-webmaker-partners-link` item on the page? But if there is one, it's a nice extra to track this too.

Comment (mmmavis) [inline public/js/pages/localweb.js:44]: This ID was added to the locale source file few hours ago (see [here](https://github.com/mmmavis/webmaker.org/blob/f0d46e07a6149651b04cddc1ce8a14826d89ad83/locale/en_US/localweb.json#L83)) It probably hasn't got uploaded to s3 yet.

**Comment by mmmavis** [comment]
Gonna merge this now.  Will let you know when I get this onto staging :) Thanks @adamlofting

**Pull request merged**